src/udev/net/link-config.c

```diff
@@ -1314,6 +1362,52 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        CPUSet *mask = NULL;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (!config->rps_cpu_mask) {
+                mask = new0(CPUSet, 1);
+                if (!mask)
+                        return log_oom();
+
+                config->rps_cpu_mask = mask;
+        }
+
+        if (isempty(rvalue)) {
+                /* Resetting cpu set but not freeing it. It's necessary to apply '0' value
+                 * or to unset previously applied 'non-zero' value. */
+                cpu_set_reset(config->rps_cpu_mask);
+                return 0;
+        }
```

Comment:
That's tricky, and conflicts with our usual way. Please make the empty string is equivalent to 'nothing specified'.
Please accept another special word e.g. `"reset"` or `"disable"` to assign `"0"` for the sysattr.